docs/src/1_Mechanics/01_PlayerGuide_Full.md

```diff
@@ -408,70 +437,79 @@ over. This ends the turn order, and characters can act independently of turn ord
-Hand. A Dealer makes a DR 3 Strength Save as a Free Action at the end of their turn to
-attempt to end the freeze.
-
-9. **Suffocating** At the beginning of their turn, a Dealer makes a DR 3 VIT Save. On
-failure, take 2 damage bypassing AP. On a success, take 1 damage.
-
-10. **Charmed** At the start of their turn or whenever taking damage, a Dealer makes a DR
-3 GUT Save. On failure, they may not take actions against the interests of the creature
-that caused the condition. Another creature may use a Major action to end this effect.
-This effects ends automatically if the target is damaged by the creature that caused
-the condition.
-
-11. **Enthralled** At the start of their turn or whenever taking damage, a Dealer makes a
-DR 3 CON Save. On failure, they must move up to their Speed and/or perform 1 Major
-Action available to them, as directed by the creature that caused the condition.
-Another creature may use a Major action to end this effect. This effects ends
-automatically if the target is damaged by the creature that caused the condition.
-
-12. **Invisible** When
-making a Check to detect an Invisible creature, draw with Lower Hand. Unless otherwise
-stated, using a Power ends the Invisible condition. The attack that ends Invisibility is
-drawn at Upper Hand.
-
-13. **Grappled** Movement speed is reduced to 0 and a Grappled creature cannot gain
-the benefit of additional speed. They have Lower Hand when using any Power. A
-creature can end the Grappled condition by making a successful Contested Brute or
-Finesse Check against target's Brute.
+  1. **Stunned** If a character is hit by a critical attack, they make a DR 3 CON
+  Save at the start of their turn. If failed, they may either move or
+  make one Action on their turn, not both. Some Powers can cause the Stunned Condition.
+
+  2. **Entangled** A character may expend a Major Action to make a DR 3 STR Save. Until they
+  are successful, their Speed is 0 and any Agility Saves are made with Lower Hand.
+
+  3. **Knocked Down** A character must expend a Minor Action to get up. Their movement speed
+  is halved, rounding up. When Knocked Down, attackers within 1 space draw with Upper Hand
+  and attackers further than 1 space away draw with Lower Hand.  All STR and AGL Saves are
+  made with Lower Hand. A character can use 1 speed to willingly go prone as part of their
+  movement, gaining the Knocked Down status.
+
+  4. **Blinded, Deafened** At the start of their turn, as a Free Action, a character makes
+  a DR 3 Intuition Save. If they fail, they draw Lower Hand (3) for any action requiring
+  the relevant sense.
+
+  5. **Knocked Out** Knocked Out (KO) characters are unable to act until treated by a Craft
+  check or a Healing Power (DR 5 vs. the KO character's TC, minus 1 for each turn passed
+  since the effect onset). If a Knocked Out Dealer is attacked further, the Player begins
+  an Epic Event alone to avoid *Death*.
```

Comment:
In my initial draft, I made no mention of Death and kept it implied for a more family friendly vibe